Directory.Packages.props

```diff
@@ -96,7 +104,7 @@
     <PackageVersion Include="System.Security.Cryptography.Pkcs" Version="9.0.0" />
     <PackageVersion Include="System.Security.Cryptography.X509Certificates" Version="4.3.2" />
     <PackageVersion Include="System.Security.Cryptography.Xml" Version="9.0.0" />
-    <PackageVersion Include="System.Text.Json" Version="9.0.0" />
+    <PackageVersion Include="System.Text.Json" Version="9.0.3" />
```

Comment:
I hope these versions have been bumped in VS too :) Are all the bumps here required to fix the problem or can any of them wait a bit until we flow the latest interactive changes into VS?